portfolio/src/main/webapp/index.html

```diff
@@ -35,33 +32,36 @@
     <div id="navbar" class="navbar-collapse">
         <ul class="nav navbar-nav">
             <li><a href="index.html">Home</a></li>
-            <li><a href="resume.pdf">Resume</a></li>
+            <li><a href="images/resume.pdf">Resume</a></li>
             <li><a href="mailto:shivani.ghanta@gmail.com">Email Me</a></li>
 
         </ul>
     </div>
 
 </div>
       </nav>
-
       <!-- Title Content & Bio -->
+
       <div class="bitmoji-top">
-              <div class="container">
-                  <div class="row">
-                      <div class="about">
-                          <div id="photos">
-                              <center><img id = "bitmoji" src="images/bitmoji.PNG" class="img-responsive" height="420" width="420"/></center>
-
-                           </div>
-                       </div>
-                       
-                       <center><div class="about">
-                           <div class="bio">
-                               <h1>Shivani Ghanta</h1>
-                               <h3>Aspiring Software Engineer</h3>
-                               <p>I’m an incoming junior majoring in Computer Science at UIUC. I'm currently an Engineering STEP Intern at Google.</p>
+		<div class="container">
+			<div class="row">
+				<div class="about">
+					<div class="bio">
+						<h1>Shivani Ghanta</h1>
+                        <h3>Aspiring Software Engineer</h3>
+                        <hr class="top-bio">
+						<br>
+                        <p>
+                        I’m an incoming junior majoring in Computer Science at the University of Illinois at Urbana-Champaign.<br>
```

Comment:
Most software development projects have a maximum number of characters per line, which is intended to keep the content of the code visible without needing to scroll to the right to read it. This helps making development and code reviews easier.

Github allows for a max of 125 visible characters in code reviews, so I would recommend to stay within that range for clarity purposes.

You can find the recommended guidelines for Google HTML projects at https://google.github.io/styleguide/htmlcssguide.html. Some noteworthy items:

- [the indentation is recommended to be 2 spaces](https://google.github.io/styleguide/htmlcssguide.html#Indentation) (I see from 4 spaces to 6 spaces in this file, which results in more easily going over the character limit.
 - [Breaking long lines is optional for HTML](https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping), but this is a good practice to follow as it applies to most languages / projects.